creation/create_cvmfsexec_distros.sh

```diff
@@ -24,7 +25,7 @@ or a comma-separated list of values from the options {osg|egi|default}.
 
 PLATFORMS_LIST (optional): indicates machine types (platform- and architecture-based)
 for which distributions is to be built. Can be empty, a single value or a
-comma-separated list of values from the options {rhel7-x86_64|rhel8-x86_64|suse15-x86_64}.
+comma-separated list of values from the options {rhel7-x86_64|rhel8-x86_64|suse15-x86_64|rhel8-aarch64|rhel8-ppc64le}.
```

Comment:
In the instructions, there should be also rhel9

Reply:
Updated as suggested.